resources/views/backend/books/create.blade.php

```diff
@@ -0,0 +1,101 @@
+                <select name="category_id" id="category">
+                </select>
+              </div>
+              <div class="form-group">
+                <label for="author">{{ __('Author') }}</label>
+                <input name="author" type="text" class="form-control" id="author" placeholder="{{ __('Enter author') }}" value="{{ old('author') }}">
+                @if($errors->first('author'))
+                  <span class="text-danger">{{ $errors->first('author') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="price">{{ __('Price') }}</label>
+                <input name="price" type="text" class="form-control" id="status" placeholder="{{ __('Enter status') }}" value="{{ old('price') }}">
+                @if($errors->first('price'))
+                  <span class="text-danger">{{ $errors->first('price') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="donate_by">{{ __('Donate By') }}</label>
+                <input name="donate_by" type="text" class="form-control" id="donate_by" placeholder="{{ __('Enter donater') }}" value="{{ old('donate_by') }}" >
+                @if($errors->first('donate_by'))
+                  <span class="text-danger">{{ $errors->first('donate_by') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="year">{{ __('Year') }}</label>
+                <input name="year" type="text" class="form-control" id="year" placeholder="{{ __('Enter year') }}" value="{{ old('year') }}">
+                @if($errors->first('year'))
+                  <span class="text-danger">{{ $errors->first('year') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="description">Description</label></br>
+                <textarea name="description" class="textarea" placeholder="{{ __('Description about this book') }}"
+                          style="width: 100%; height: 200px; font-size: 14px; line-height: 18px; border: 1px solid #dddddd; padding: 10px;"></textarea>
+              </div>
+              <div class="form-group">
+                <label for="exampleInputFile">{{ __('Book Image') }}</label>
+                <input name="image" type="file" id="exampleInputFile">
+                <p class="help-block">{{ __('Only upload image with maximum 10mb') }}</p>
+                @if($errors->first('image'))
+                  <span class="text-danger">{{ $errors->first('image') }}</span>
+                @endif
+              </div>
+
+              <div class="box-footer">
+                <button id="btn-add-book" type="submit" class="btn btn-primary">{{ __('Submit') }}</button>
+                <button type="reset" class="btn btn-warning">{{ __('Clear') }}</button>
+              </div>
```

Comment:
I think u should put back-button to come back previous url and Clear is replaced Reset